### ISSUE TITLE
Comments part of robot file lost when updating

### ISSUE THREAD

**Issue by Harm10** [robotframework/RIDE#2393]
I am using Comments in some robot files like described in [https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#id563](url) section 2.1.2
As soon as I update such a robot file through RIDE that contains Comments the whole part is removed.
It should remain untouched. Can this be fixed?
I see this on RIDE v2.0b2.dev3 running on Python 3.7.9. but also on RIDE 1.5.2.1 on Python 2.7.13.

If fixed can the code fix be published here so I can also fix 1.5.2.1?

**Comment by HelioGuilherme66** [comment]
This is known since 2015, on issue #1533 and planned to fix on #1984 

Why would you want to use 1.5.2.1, when we have newer 1.7.4.2 version?

**Comment by Harm10** [comment]
Sorry I did not check the known issue list...................... my bad!

Does version 1.7.4.2 run on Python 2.7.13? On that platform I cannot upgrade the Python version yet (to my dismay I should add) and cannot experiment either...............

**Comment by HelioGuilherme66** [comment]
Yes, it is the last version to support Python 2.7. See the release notes, [here](https://github.com/robotframework/RIDE/blob/master/doc/releasenotes/ride-1.7.4.2.rst). Very important is the change from **: FOR** to **FOR END**. 
Take care on the upgrade, should you need to downgrade, you may lose some settings.

**Comment by Harm10** [comment]
Thanks for the information. I noticed this change on the : FOR syntax some months ago. That is a very heavy change for me/us to do as we use FOR logic a lot. So I am afraid to do these changes. It is on the to do list as is the upgrade from Python 2.7 which is ancient (I inherited this set-up alas).
